packages/SwingSet/src/devices/timer/device-timer.js

```diff
@@ -132,33 +149,28 @@ function makeTimerMap(state = undefined) {
   }
 
   // We don't expect this to be called often, so we don't optimize for it.
-  function remove(targetHandler) {
+  /**
+   *
+   * @param {Waker} target
```

Comment:
Please keep the old name for this parameter; it's more clear. 
```suggestion
   * @param {Waker} targetHandler
```